shared/api/server.go

```diff
@@ -157,8 +157,8 @@ type ServerStorageDriverInfo struct {
 // swagger:model
 type ServerPut struct {
 	// Server configuration map (refer to doc/server.md)
-	// Example: {"core.https_address": ":8443", "core.trust_password": "xyz"}
-	Config map[string]string `json:"config" yaml:"config"`
+	// Example: {"core.https_address": ":8443", "core.trust_password": true}
```

Comment:
why would core.trust_password be `true` now you've kept the removal of hidden fields concept?

Reply:
Ah thanks, this one I have missed.

Reply:
fixed